server/dockerCompose/README.md

```diff
@@ -0,0 +1,20 @@
+# Docker compose setup for patavi
+
+This docker compose setup is meant to provide an easier way to get patavi up and running on a personal computer or laptop. It is meant to be used in conjunction with the docker compose setup of our other modules (ADDIS, GeMTC, MCDA).
+
```

Comment:
"NB: ** all services will be run with simple development/testing passwords. This is intended as a local, offline environment. For deployment in secure/online environments, apply different methodology and better security, or contact the developers for advice."